### DISCUSSION TITLE
Accessing older versions of packages

### DISCUSSION THREAD

**Discussion by ingluisjimenez** [oerdnj/deb.sury.org#1824] [General]
**Frequently asked questions**

- [x] I have read [Frequenty Asked Questions](https://github.com/oerdnj/deb.sury.org/wiki/Frequently-Asked-Questions)

**Is your feature request related to a problem? Please describe.**
I would like to know if there is a way to access older versions of some of the packages, specifically PHP. I am trying to downgrade an application to PHP 8.0.11, but wasn't able to find a mirror containing that specific minor version. 

**Describe the solution you'd like**
Access a mirror with older builds of the packages, or either build the specific version. This seems to be the build from last year when that specific version was built. https://launchpad.net/~ondrej/+archive/ubuntu/php/+build/22109351

**Describe alternatives you've considered**
Tried searching for a mirror for the specific version but couldn't locate one.

**Distribution (please complete the following information):**
 - OS: Ubuntu
 - Architecture: amd64
 - Repository: packages.sury.org

**Package(s) (please complete the following information):**
php

**Additional context**
N/A

**Comment by mlebrun** [comment]
Is there any solution to this?

**Reply by oerdnj** [reply]
This is not the purpose of this repository. But you can always compile the version you need yourself.

**Reply by gazben** [reply]
Where can I find the build script for your versions so I compile it myself?
